resources/carrier/en/374.txt

```diff
@@ -15,15 +15,15 @@
 # Data from Wikipedia:
 # http://en.wikipedia.org/wiki/Telephone_numbers_in_Armenia#Mobile_network_codes
 
-37441|Orange
+37441|Ucom
```

Comment:
Here's the official Orange's publication: http://www.orange.com/en/Press-and-medias/press-releases-2016/press-releases-2015/Orange-signs-an-agreement-with-Ucom-for-the-sale-of-100-of-its-shares-in-Orange-Armenia